app/controllers/Application.scala

```diff
@@ -95,29 +100,29 @@ class Application @Inject()(repo: Repo, app: play.api.Application, val messagesA
   }
 }
 
-case class ProbForm(prob: String)
+case class SolutionForm(solution: String)
 
-case class AddProbForm(task: String, blank: String, test: String)
+case class AddTaskForm(taskDescription: String, solutionTemplate: String, referenceSolution: String)
 
 object Application {
   val logoutDone = "Logout done"
-  val prob = "prob"
+  val solution = "solution"
```

Comment:
names and values of these variables are a bit confusing. Why do we need them at all?